pkg/trace/obfuscate/sql.go

```diff
@@ -189,9 +226,7 @@ func obfuscateSQLString(in string) (string, error) {
 		}
 		lastToken = token
 	}
-	if out.Len() == 0 {
-		return "", errors.New("result is empty")
-	}
+
```

Comment:
Also this line please. In Go I found it was uncommon to have many "paragraphs" and empty lines. Plus, the closing curly braces already provide spacing.